README.md

```diff
@@ -32,14 +32,25 @@ Cumulus.
 
 ## Development Setup
 
-You can run tests and deploy the stack inside of a Docker container:
+Set DAAC_DIR as an enviroment var:
 
-        $ make image
-        $ DAAC_DIR=$HOME/projects/acme-cumulus make container-shell
+      $ export DAAC_DIR=$HOME/projects/acme-cumulus
 
 Here DAAC_DIR is the absolute path to the fork of `CIRRUS-DAAC` that
 you would like to deploy.
 
+If DAAC_DIR is not set, make will give you the following error:
```

Comment:
The root of this error is the addition of [force](https://github.com/asfadmin/CIRRUS-core/blob/3b0f19c29200bb951826d8da7db10e5405c9b23e/Makefile#L323)